graph/src/data/subgraph/mod.rs

```diff
@@ -271,6 +273,11 @@ pub enum SubgraphRegistrarError {
     NameNotFound(String),
     #[fail(display = "Ethereum network not supported by registrar: {}", _0)]
     NetworkNotSupported(String),
+    #[fail(
+        display = "Ethereum network {} not found with capabilities: {:?}",
```

Comment:
I'm not sure I'd say `network {} not found`. Maybe something like `Ethereum nodes for network {} are missing the following capabilities: {}`?